packages/runtime/src/widgets/Search.tsx

```diff
@@ -121,7 +117,7 @@ export const Search: React.FC<SearchProps> = ({
         onSearchAction.callback({ search: searchValue });
       }
     },
-    onSearch?.debounceMs,
+    onSearch?.debounceMs || 500,
```

Comment:
I think we should keep this 0 by default.

Reply:
okay, let me set it to 0.